eng/Directory.Build.Common.props

```diff
@@ -121,6 +121,7 @@
   <ItemGroup Condition="'$(IsMgmtLibrary)' == 'true'">
```

Comment:
nit: While having these packages references in the common file is convenient from some aspects there is also value in having them directly in each project file. For dependencies it is often nice to be able to see them directly in the csproj file itself which is why Azure.Core is in each of the projects for our track 2 data plane libraries. I leave that up to you but you should consider putting these in the template project so they end up in each project instead of being in the shared props.